wordpress-mobile/WordPress-iOS
# QR Login: Adds Login Menu Item and base structure

| 📚 Project: #18594 | 
|:---:|

### Description
This adds a base structure for the QR Login process, the FeatureFlag/AppConfiguration, and the Scan Login Code item to the Me Menu.

Currently tapping the item in the Me menu doesn't do anything.

### Screenshots
<img src="https://user-images.githubusercontent.com/793774/170767560-c6950f34-fe42-4182-b408-7706e4ce54f0.PNG" width="160" />

### To test:

1. Build and launch the app
2. Go to the My Site tab
3. Tap on your profile icon in the top right corner of the view
4. 👁️ Notice there is a new item for Scan Login Code

### Regression Notes
1. Potential unintended areas of impact
None

5. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Testing

7. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
